config/crd/bases/velero.io_volumesnapshotlocations.yaml

```diff
@@ -1,8 +1,10 @@
+
+---
```

Comment:
Assuming make bundle worked, but just wanted to confirm operator-sdk was happy with this